larsim/Utils/SCEOffsetBounds.cxx

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file larsim/Utils/SCEOffsetBounds.cxx
+ *
+ * @brief Implementation of SCE Offset Bounds Checking method(s)
+ *
+ * @author Tom Junk (trj@fnal.gov)
+ *
+ * $Log: $
+ */
+
+// LArSoft
+#include "larsim/Utils/SCEOffsetBounds.h"
+
+bool larsim::Utils::SCE::out_of_bounds(geo::Vector_t const &offset)
+{
+  constexpr double limit{1.e9};
```

Comment:
This suggestion really doesn't need to hold up the PR (and this suggestion is basically purely stylistic due to there being only one function in the implementation file) but this declaration --could-- be moved outside of the function.

Reply:
Yes, if we need to expose that limit number elsewhere we could define it there.  The intended use is in fact for code in dunetpc to set space charge offsets that exceed the limit so that charge is dropped in the simulation, and making sure that the large value designed to exceed the limit will continue to do so in the future is a reason to expose that declaration somewhere else, but I don't see the need for this number to change.